### PR TITLE
Account login

### PR DESCRIPTION
**Pull request by mashish584**

### CODE REVIEW

File: server/src/resolvers/mutation.ts [hunk 1]
```diff
@@ -66,6 +66,44 @@ const Mutation = {
...
+            verificationCode: null,
+            isActive: true,
+          },
+        });
+        return { success: true };
+      }
+
+      return { success: false };
+    } catch (err) {
+      throw new Error("Server Error");
```
Comment (ilakshay14) [inline server/src/resolvers/mutation.ts:109]: would suggest to throw more appropriate/better readable errors like
user not found
authentication error if token did not verify, etc.

File: server/src/schema.graphql [hunk 2]
```diff
@@ -4,12 +4,14 @@ type Query {
 
 type Mutation {
   createUser(data: CreateUserInput): SignupPayload!
+  verifyUserEmail(username: String!, token: String!): VerificationPayload!
```
Comment (ilakshay14) [inline server/src/schema.graphql:7]: for one parameter do you need to create a separate input type? or are you planning to add more properties.

{ success: Boolean }

**Review by ilakshay14 — COMMENTED** [review]
Added few comments. Have a looks. No issues from the functionality perspectives.

File: server/src/resolvers/mutation.ts (hunk 1 above)
Comment (mashish584) [reply server/src/resolvers/mutation.ts:109]: Error are mostly unexpected one in this case. 

Example: DB is not responding or something wrong in backend logic.

File: server/src/schema.graphql (hunk 2 above)
Comment (mashish584) [reply server/src/schema.graphql:7]: I prefer this way as verification payload might be useful in future resolvers as well.

**Pull request merged**